test/inproc.js

```diff
@@ -10,7 +10,7 @@ test('inproc socket pub sub', function (t) {
     t.plan(1);
 
     var pub = nano.socket('pub');
-    var sub = nano.socket('sub');
+    var sub = nano.socket('sub', {chan:''});
```

Comment:
While it's nice to see you're being explicit about not subscribing to any chans, this makes it look like the second arg to nano.socket is required.  Let's remove the explicitly empty options argument, that way users reading the tests for usage don't think that they are required to construct a meaningless object.